miio/vacuum_cli.py

```diff
@@ -231,6 +231,13 @@ def manual(vac: miio.Vacuum):
     # if not vac.manual_mode and command :
 
 
+@manual.command()
+@pass_dev
+def tui(vac: miio.Vacuum):
+    """TUI for the manual mode."""
+    miio.VacuumTUI(vac).run()
```

Comment:
Wrt the previous comment on curses import errors, this would be:
```
from miio.vacuum_tui import VacuumTUI
try:
    tui = VacuumTUI(vac)
    tui.run()
except Exception as ex:
    click.echo("Error during TUI: %s" % ex)
```